packages/@react-aria/list/src/useListItem.ts

```diff
@@ -166,12 +166,15 @@ export function useListItem<T>(props: AriaListItemOptions, state: ListState<T>,
     }
   };
 
+  let descriptionId = useSlotId();
```

Comment:
You'll want to return this in a `descriptionProps` much like https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/listbox/src/useOption.ts#L156-L161. I've tried doing this locally but having difficulties getting it to announce anything but the aria-label, changing it to aria-describedby + aria-labelledby doesn't seem to work that well either (still only announces the aria-labelledby. 

Will need to experiement which a labelling strategy that announces properly

Reply:
Maybe a single aria-labelledby that has the ids of the content + description?

Reply:
Also now that I look at it, not sure why we have a slot for content and for text?

Reply:
I'm seeing the same in VO, lets take a look later today and see what the best solution is

Reply:
Resolved, see https://github.com/adobe/react-spectrum/pull/3121#issuecomment-1138863077